src/wre/game.ts

```diff
@@ -36,11 +36,15 @@ const game: Natives = {
       const x = position.get('x')?.innerNumber
       const y = position.get('y')?.innerNumber
 
+
       if(x != undefined && y != undefined) {
         const roundedX = round(x)
         const roundedY = round(y)
         for(const visual of visuals.innerCollection!) {
-          const otherPosition = yield* this.send('position', visual)
+          const metodo = visual.module.lookupMethod('position', 0)!
+
+          const otherPosition = metodo.isSynthetic ? visual.get('position') :yield* this.send('position', visual)
```

Comment:
Acá se podría dejar un comentario sobre la optimización que se hace.
También conviene usar `invoke` en vez del send, aprovechando que ya tenemos el método.

Y ojo con el english.

Sería
```suggestion

          // Every visual understand position(), it is checked in addVisual(visual).
          // Avoid to invoke method position() for optimisation reasons. 
          //    -> If method isSynthetic then it is a getter, we can access to the field directly
          const method = visual.module.lookupMethod('position', 0)!
          const otherPosition = metodo.isSynthetic ? visual.get('position') :yield* this.invoke(method, visual)
```